### DISCUSSION TITLE
Hello tonivade, I have a question.

### DISCUSSION THREAD

**Discussion by ygger** [tonivade/claudb#157] [Q&A] [answered]
In the class ClauDB in the package com.github.tonivade.claudb, what is the purpose of super.start() in the start() method? It seems that RespServerContext's start() is empty.Hope you can clarify.

**Comment by tonivade** [comment, marked as answer]
Hi @ygger. Thank you for your interest in claudb.

RespServerContext.start() method is a hook to allow implementations of RespServerContext to have a proper initialization. This way ClauDB class can be initialized from the framework.

The call to super.start() can be removed but it's a good practice to call the parent method. This way if RespServerContext class adds something to the start method the implementation will execute that code also.

**Reply by ygger** [reply]
Thank you very much for your response. Your answer has helped me unravel my confusion.
